vendor/knative.dev/pkg/injection/sharedmain/main.go

```diff
@@ -226,6 +226,10 @@ func MainWithConfig(ctx context.Context, component string, cfg *rest.Config, cto
 	profilingServer := profiling.NewServer(profilingHandler)
 
 	CheckK8sClientMinimumVersionOrDie(ctx, logger)
+	// HACK: should go away when we move away from < 4.11 releases
```

Comment:
Do we want that to be in a patch as well (if we e.g. update deps somehow?).

Reply:
1.9 has part of the fix there. This is where I will create the patch and that would be the diff also to use for main.